packages/manager/modules/hub/src/components/billing-summary/template.html

```diff
@@ -0,0 +1,35 @@
+<div class="manager-hub-billing-summary">
+    <h3 data-translate="hub_billing_summary_title"></h3>
+    <oui-select
```

Comment:
don't you think loading is required when bills and other data are being fetched?

Reply:
yes, i'm not sure how we should handle loading in the hub tiles though. oui-skeleton doesn't looks good with the current tile style & color